quickstarts/microsoft.attestation/attestation-provider-create/main.bicep

```diff
@@ -0,0 +1,29 @@
+@description('Name of the Attestation provider. Must be between 3 and 24 characters in length and use numbers and lower-case letters only.')
+param attestationProviderName string = uniqueString(resourceGroup().name, deployment().name)
```

Comment:
let's remove deployment().name from the seed, it will create idempotency challenges as is...